matchzoo/models/bimpm.py

```diff
@@ -0,0 +1,420 @@
+"""An implementation of BiMPM Model."""
```

Comment:
Please add this mode in the `__init__.py`, and please run and test it on a specific dataset.